spec/components/alert_icon_component_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe AlertIconComponent, type: :component do
+  let(:icon_name) { nil }
+  let(:tag_options) { {} }
+
+  subject(:rendered) do
+    render_inline(described_class.new)
+  end
```

Comment:
It's a bit strange to me that we define the `subject` but in all except the first test case, we recreate it with a local variable assignment. I'd think we should be consistent one way or the other: either always use the subject (and configure using `context`-ual `let`s) or remove the subject and always assign in the test case.

Reply:
Not strange, just yours truly still trying to figure out rspec!